src/main/scala/io/iohk/ethereum/network/handshaker/EtcHelloExchangeState.scala

```diff
@@ -31,18 +31,15 @@ case class EtcHelloExchangeState(handshakerConfiguration: EtcHandshakerConfigura
     log.debug("Protocol handshake finished with peer ({})", hello)
     // FIXME in principle this should be already negotiated
     Capability.negotiate(hello.capabilities.toList, handshakerConfiguration.blockchainConfig.capabilities) match {
-      case Some(ProtocolVersions.ETC64) => {
+      case Some(ProtocolVersions.ETC64) =>
         log.debug("Negotiated protocol version with client {} is etc/64", hello.clientId)
         EtcNodeStatus64ExchangeState(handshakerConfiguration)
-      }
-      case Some(ProtocolVersions.ETH63) => {
+      case Some(ProtocolVersions.ETH63) =>
         log.debug("Negotiated protocol version with client {} is eth/63", hello.clientId)
         EthNodeStatus63ExchangeState(handshakerConfiguration)
-      }
-      case Some(ProtocolVersions.ETH64) => {
+      case Some(ProtocolVersions.ETH64) =>
         log.debug("Negotiated protocol version with client {} is eth/64", hello.clientId)
         EthNodeStatus64ExchangeState(handshakerConfiguration)
```

Comment:
could you also fix the 'orElse' clause that I introduced (`orElse(WireDecoder)`)

Reply:
I would like to do this in a follow up PR, this one has become quite old and heavy